src/controllers/post.js

```diff
@@ -152,13 +152,20 @@ export const deletePost = async (req, res) => {
         'Request authorization to delete post'
       )
     }
+
+    const deleteComments = await dbClient.comment.deleteMany({
+      where: {
+        postId: id
+      }
+    })
+
     const deletePost = await dbClient.post.delete({
       where: {
         id
       }
     })
 
-    return sendDataResponse(res, 201, deletePost)
+    return sendDataResponse(res, 201, deleteComments, deletePost)
```

Comment:
What are you trying to do on this line? You're passing 4 arguments until a function that only accepts and uses 3 parameters.

Reply:
Hi, I'm trying to first delete the comments within a post and then to delete the post itself using cascading deletes from Prisma CRUD

Reply:
Okay. I'm just not sure why you are sending both `deleteComments` and `deletePost` to `sendDataResponse`. `sendDataResponse` will send the `deleteComments` variable to the user, but not the `deletePost`. Do we need to send the `deletePost` variable as well?

Reply:
Ok, I've understood:  ````return sendDataResponse(res, 201, { deletedPost, deletedComments })````
```